static/compiler.js

```diff
@@ -96,7 +99,30 @@ define(function (require) {
         this.outputEditor = monaco.editor.create(this.domRoot.find(".monaco-placeholder")[0], {
             scrollBeyondLastLine: false,
             readOnly: true,
-            language: 'asm'
+            language: 'asm',
+            glyphMargin: true
+        });
+
+        this.outputEditor.addAction({
+            id: 'viewsource',
+            label: 'View source',
+            keybindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.F10],
+            keybindingContext: null,
+            contextMenuGroupId: 'navigation',
+            contextMenuOrder: 1.5,
+            run: function(ed) {
+                var desiredLine = ed.getPosition().lineNumber - 1;
+                self.eventHub.emit('editorSelectLine', self.sourceEditorId, self.assembly[desiredLine].source);
+            }
+        });
+
+        this.outputEditor.onMouseMove(function (e) {
+            setTimeout(function() {
```

Comment:
@mattgodbolt What do you think about this timeout as to throttle the MouseMove event and not comput it every "tick"? There is also one on the other text editor